update/ansible/playbook/tasks/roles/nginx/files/ssl/generate-ssl-certificate

```diff
@@ -3,6 +3,16 @@
 set -o errexit
 
 mkdir -p /srv/nginx
+if [ ! -w "/srv/nginx" ]; then
+    echo "Directory /srv/nginx is not writable";
+    if [ ! -e /srv/nginx/certificate.key -o ! -e /srv/nginx/certificate.crt ]; then
+        echo "Can't generate self-signed certificate. Exiting...";
+        exit 1;
+    else
+        echo "Using existing self-signed certificate";
+        exit 0;
```

Comment:
We exit here without checking for the presence of two files only - `certificate.key` and `certificate.crt`. However, in `/etc/nginx/conf.d/pmm.conf` nginx is expecting the `dhparam.pem` file, which is supposed to be copied (lines 17..19 in this file) but won't be.

I wonder if nginx still work in such a case?

Reply:
Need to check, but it makes no sense to copy them from container if certificate.key and certificate.crt is passed. So no difference if it will fail here or there. Check was added there to not fail if any file exist there.

Reply:
I was referring to clean exit with `exit 0` - the file won't be present.